bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/productcarousel/ProductCarousel.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Adobe.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.commerce.core.components.models.productcarousel;
+
+
+import com.adobe.cq.commerce.core.components.models.productlist.ProductListItem;
+import java.util.Collection;
+import javax.annotation.Nonnull;
+import org.osgi.annotation.versioning.ProviderType;
+
+@ProviderType
+public interface ProductCarousel {
+    
+    /**
+     * returns the product list  for Product Carousel
+     * @return {@link Collection} of {@link ProductListItem}s
+     */
+    @Nonnull
+    default Collection<ProductListItem> getProducts() {
```

Comment:
Please return List<ProductListItem> to emphasize that it's an order collection.

Reply:
Return type is changed to List interface